R/track-util.R

```diff
@@ -0,0 +1,142 @@
+#' track helpers
```

Comment:
@0x00b1 I guided @cells2numbers in that direction, i.e., to use `*-utils.R` but did so because I saw it elsewhere (dplyr). I see the benefit in keeping it in one place (track.R) but wondering if that would make it too cluttered?